src/main/protobuf/soc/gameState.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto2";
+
+package protos.soc;
+
+import "soc/board.proto";
+import "soc/inventory.proto";
+import "soc/playerState.proto";
+
+message GameState {
+    required Board board = 1;
+    repeated PlayerState playerStates = 2;
+    required int32 developmentCardsInDeck = 3;
+    required int32 currentTurn = 4;
+    required bool canRollDice = 5;
+    required bool canPlayCard = 6;
+    required ResourceSet bank = 7;
+    optional DevelopmentCardSet developmentCardDeck = 8;
```

Comment:
Give that development card set is just a list of development cards, why not `repeated DevelopmentCard`... Same for `ResourceSet`

Reply:
Also, in the case of the development card deck, why not just an integer count ie. `remainingDevelopmentCardCount`

For the resources, what was your thought process in repeated values, vs a map? (I know if my PR I've used repeated values--I think there can be a use case for both, but again, we should define a methodology / reasoning behind it)

Reply:
You cant use enums as keys for a map in grpc. Very annoying. Repeated value was the best alternative I could think of and am still thinking of better alternatives

Reply:
Ugh. Really annoying. Yeah, we should think more.